protocol-definitions/Jet.yaml

```diff
@@ -286,3 +286,18 @@ methods:
           nullable: false
           since: 2.0
           doc: ''
+  - id: 15
+    name: getJobAndSqlSummaryList
+    since: 2.5
+    doc: ''
+    request:
+      retryable: true
+      partitionIdentifier: -1
+      params: []
+    response:
+      params:
+        - name: response
+          type: HazelcastJsonValue
```

Comment:
As far as I understand, this actually should be `List_JobAndSqlSummary`, where `JobAndSqlSummary` would be a new custom type definition. 

I don't think it is okay to return the response as JSON here. It might be flexible and easy to use, but we need to maintain backward/forward compatibility of the protocol through this repository, and JSON could change anytime without us noticing anything

Reply:
Done